system/config/countries.php

```diff
@@ -215,14 +215,16 @@
 	'sh' => 'Saint Helena',
 	'si' => 'Slovenia',
 	'sj' => 'Svalbard and Jan Mayen Islands',
-	'sk' => 'Slovak Republic',
+	'sk' => 'Slovakia',
 	'sl' => 'Sierra Leone',
 	'sm' => 'San Marino',
 	'sn' => 'Senegal',
 	'so' => 'Somalia',
 	'sr' => 'Suriname',
-	'st' => 'Sao Tome and Principe',
+	'ss' => 'South Sudan',
+	'st' => 'São Tomé and Príncipe',
```

Comment:
Same as in https://github.com/contao/core/pull/7918/files#r34468434. We should stick to ASCII.